config/settings.yml

```diff
@@ -54,10 +54,14 @@ preassembly:
   bundle_directory: '/dor/staging/integration-tests/files-reaccessioning-test'
   ocr_bundle_directory: '/dor/staging/integration-tests/ocr-test'
   ocr_document_bundle_directory: '/dor/staging/integration-tests/ocr-test-document'
+  speech_to_text_bundle_directory: '/dor/staging/integration-tests/speech-to-text-test'
   hfs_bundle_directory: '/dor/staging/integration-tests/hierarchical-file-test'
   gis_bundle_directory: '/dor/staging/integration-tests/gis-test'
 
 earthworks_url: 'https://earthworks-stage.stanford.edu/catalog'
 
 ocr:
   enabled: true
+
+speech_to_text:
+  enabled: false
```

Comment:
you can override this to `true` in `config/settings/qa.local.yml` or `config/settings/stage.local.yml` to run the test on your laptop when you are confident the correct branches of pre-asembly and common-accessioning are deployed

eventually will become true for everyone when in production